apps/server/src/modules/video-conference/uc/video-conference-create.uc.spec.ts

```diff
@@ -0,0 +1,206 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { UserService } from '@src/modules/user';
+import { userDoFactory } from '@shared/testing';
+import { UserDO, VideoConferenceScope } from '@shared/domain';
+import { ObjectId } from 'bson';
+import { ForbiddenException } from '@nestjs/common';
+import { VideoConferenceCreateUc } from './video-conference-create.uc';
+import { BBBService, VideoConferenceService } from '../service';
+import { VideoConferenceOptions } from '../interface';
+import { BBBCreateResponse, BBBMeetingInfoResponse, BBBResponse, BBBRole, BBBStatus } from '../bbb';
+import { IScopeInfo, ScopeRef } from './dto';
+import { ErrorStatus } from '../error/error-status.enum';
+
+describe('VideoConferenceCreateUc', () => {
+	let uc: VideoConferenceCreateUc;
+	let bbbService: DeepMocked<BBBService>;
+	let userService: DeepMocked<UserService>;
+	let videoConferenceService: DeepMocked<VideoConferenceService>;
+
+	beforeEach(async () => {
```

Comment:
```suggestion
	beforeAll(async () => {
```
https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html

Reply:
Please check all your tests

afterAll(async () => {
    await module.close();
});

afterEach(() => {
    jest.resetAllMocks();
})

Reply:
Oh yes, thanks :)

Reply:
beforeAll please